models/user_mail.go

```diff
@@ -71,8 +71,8 @@ func GetEmailAddresses(uid int64) ([]*EmailAddress, error) {
 // GetEmailAddressByID gets a user's email address by ID
 func GetEmailAddressByID(uid, id int64) (*EmailAddress, error) {
```

Comment:
if we get it by ID why we ned the uid?

Reply:
because in same cases id is provided from query and we need to check that this email address belongs to authorized user

Reply:
Only think this was **miss used** for permissiin chek - we schould do this elsewhere

Reply:
To confirm `uid` is the owner of the record with the `id`.

Reply:
this will be another refactor ... but out of scope of this pull :)